cypress/utils/data_utils.ts

```diff
@@ -112,21 +112,20 @@ export function getDefaultTagCategories(): string[] {
 export function getRandomCredentialsData(
     type: string,
     userCred?: string,
-    gitTestingUser = false,
+    useRealUser = false,
     url?: string
 ): CredentialsData {
-    let password;
-    let user;
-
-    if (gitTestingUser) {
-        user = Cypress.env("git_user");
-        password = Cypress.env("git_password");
-    } else {
-        user = getRandomWord(6);
-        password = getRandomWord(6);
-    }
+    let password = getRandomWord(6);
+    let user = getRandomWord(6);
+    let email = getEmail();
+    //TODO: This value is set to 20 to avoid a bug. Need to be updated to 200 when bug is fixed
+    let token = getRandomWord(20);
 
     if (type === CredentialType.proxy) {
+        if (useRealUser) {
+            user = "redhat";
+            password = "redhat";
```

Comment:
Do we have a testAccount for proxy ? 
Why can't we test with random names ?

Reply:
Cause those are real creds of out internal proxy, if we want to use proxy with auth - we need to provide them. It is not sensitive data cause this proxy is located behind the firewall and can't be accessed without VPN.